src/panels/PanelBase.js

```diff
@@ -121,6 +121,22 @@ export default {
       window.open(this.popupUrl);
     },
     setMaxHeight() {
+      // Don't play the transition for this case as the loading should feel 'instant'
+      if (this.expandedBool) {
+        this.$refs.panel.style.maxHeight = 'max-content';
+        return;
+      }
+      
+      /*
+      Otherwise, since the vue transition is dependent on localExpanded, we have to manually
+      set our own transition end handlers here for the initial loading of the content.
```

Comment:
Context:
vue uses `v-show/v-if` to detect transition changes, and there is no way to explicitly trigger the vue animation process.
Hence, the additions here manually trigger the new `afterExpand` hook, the only difference being that the event listener is removed once done.

For the above however ( `this.expandedBool` ), we want to loading to feel 'instant' ( even though it isn't ), hence there's no animation there